PBMakefileGenerator.m

```diff
@@ -379,6 +380,7 @@ - (PBMakefileGenerator *) initWithProject: (PBPbxProject *) aProject;
 - (void) dealloc
 {
   RELEASE(project);
+  RELEASE(pbxbuildDir);
```

Comment:
This releases the pbxbuildDir but not the pbxcriptsDir.

Reply:
True. Let me fix that.  The pbxbuild only runs through then exits so it's not that big of a deal with the application itself, but for reuse it needs to be fixed.

Reply:
Plus, if we want to merge our work back to the master branch of GNUstep, probably good we have it nailed down to avoid memory leaks.